scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -435,3 +361,46 @@ task GetSampleIds {
     File gvs_ids = "gvs_ids"
   }
 }
+
+task CurateInputLists {
+  input {
+    String dataset_name
+    String project_id
+    File input_vcf_index_list
+    File input_vcf_list
+    File input_samples_to_be_loaded_map
+    File input_sample_name_list
+
+    String? service_account_json_path
+  }
+
+  String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
+  command <<<
+    set -ex
+    if [ ~{has_service_account_file} = 'true' ]; then
+      gsutil cp ~{service_account_json_path} local.service_account.json
+      gcloud auth activate-service-account --key-file=local.service_account.json
+    fi
+
+    gsutil cp  ~{input_vcf_index_list} input_vcf_index_list_file
+    gsutil cp  ~{input_vcf_list} input_vcf_list_file
+    gsutil cp  ~{input_samples_to_be_loaded_map} input_samples_to_be_loaded_map_file
+    gsutil cp  ~{input_sample_name_list} input_sample_name_list_file
+
+    python3 /app/curate_input_array_files.py
+  >>>
+  runtime {
+    docker: "us.gcr.io/broad-dsde-methods/variantstore:rsa_skip_samples_20220602"
+    memory: "3 GB"
+    disks: "local-disk 100 HDD"
+    bootDiskSizeGb: 15
+    preemptible: 0
```

Comment:
Can this be removed? (that is, is 'preemptible: 0' the same as not including it? (stupid wdl question).

Reply:
Not stupid at all; I think this was a copy/paste issue.  No reason to have this be non-preemptible.  Good catch!